client/src/shared/style/Home.css

```diff
@@ -0,0 +1,224 @@
+body {
```

Comment:
There's a lot in this css file, and at first glance I can't tell what is actually shared shared across pages and is specific to the Home component.

I recommend putting _shared_ styles used throughout the site in App.scss, and only putting component-specific styles in corresponding named component s/css files. I'd also just put them with the same name in the same folder as the component so that they're easier to find.